knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: .NET MAUI Editor inside ScrollView
+description: Learn how to scroll the .NET MAUI Editor when in a ScrollView and scrolling the Editor's text when Editor is wrapped in a Border.
+type: how-to
+page_title: How to Scroll the Editor in a ScrollView - Editor for .NET MAUI
+slug: dotnet-maui-editor-scroll-inside-scrollview
+tags: editor, scrollview, scrolling, .NET MAUI, multiline entry
+res_type: kb
+---
+
+## Environment
+
+| Product |
+| --- |
+| Editor for .NET MAUI |
+
+## Description
+
+I want the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-8.0" target="_blank">`Editor` for .NET MAUI</a> to scroll on Android when added inside a `ScrollView`.
+
+This KB article also answers the following questions:
+- How to use the .NET MAUI `Editor`, so the editor to scroll when having more input elements on the page?
+- How to use the .NET MAUI `Editor`, so the editor to resize to accommodate new input?
+- How to make the scenarios described above to work correctly when wrapping the .NET MAUI `Editor` inside a Border?
+
+## Solution
+
+To scroll the .NET MAUI `Editor` on Android when the editor is inside `ScrollView` you need to use the editor handler to access the native Android control. Then call the  `RequestDisallowInterceptTouchEvent` with parameter `false` to the parent view the editor is placed to. 
+
+Use the Telerik .NET MAUI [RadBorder]({%slug border-overview%}) to achieve the following:
+
+* `Editor` inside `RadBorder` that resizes when content changes.
```

Comment:
Why is this a separate list and doesn't expand on the steps described above?

Reply:
so the first case is for android, and the second case when using the border is on iOS. We described mainly how to fix issues related in the maui editor using our border control.